phac-nml/irida-next
# Analysis Exports: Error when including non-completed workflow executions

## What does this PR do and why?
Currently through the UI, users are able to select any workflow execution(s) and start an export  from a respective workflow executions' index page (either personal or through project). To prevent this, validation has now been added to the `DataExports::CreateService` to check for any non-completed workflows, and will return a error flash message if one was selected.

A possible follow-up PR will give a user a warning in the create export dialog, and allow them to continue with submission and filter out the non-completed workflow executions.

## Screenshots or screen recordings
![image](https://github.com/user-attachments/assets/f7fdb865-e866-475b-85b6-22135272d5ad)

## How to set up and validate locally
From both user and project workflow execution index page:
1. Create a new workflow execution
2. Cancel the workflow
3. Select the cancelled workflow and create an export
4. Submit the export
5. You should receive an error flash message and the export should not have created

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
